GCTC-NTGC/gc-digital-talent
# TC Report - Fix page title value

Resolves https://github.com/GCTC-NTGC/gc-digital-talent/issues/3136.

This branch downstreams the changes from https://github.com/GCTC-NTGC/tc-report/pull/5

Suggested test steps

1.  Visit the tc-report site locally
2. Navigate to the report page (/en/talent-cloud/report)
3. View the page title which should no longer have a `<br>` tag as part of its value
4. It should work the same way in French, too.
